trie/snapshotTrieStorageManager.go

```diff
@@ -43,12 +43,12 @@ func (stsm *snapshotTrieStorageManager) Get(key []byte) ([]byte, error) {
 	if core.IsClosingError(err) {
 		return nil, err
 	}
-	if len(val) != 0 {
-		stsm.putInPreviousStorerIfAbsent(key, val, epoch)
-		return val, nil
+	if len(val) == 0 {
+		return nil, ErrKeyNotFound
 	}
 
-	return stsm.getFromOtherStorers(key)
+	stsm.putInPreviousStorerIfAbsent(key, val, epoch)
```

Comment:
is this change ok? On the old code we called `stsm.getFromOtherStorers(key)`

Reply:
It is ok. If the value was not found, it would have tried to get it from the checkpoints storer. But since we have removed the checkpoints storer, I also removed the `stsm.getFromOtherStorers(key)`.